woocommerce/woocommerce-android
# Issue/3700 remove combined attr

Closes #3700 - Prior to this PR, `AddAttributeFragment` had its own adapter which relied on a `CombinedAttributeModel`. These turn out to be unnecessary and we can use the same adapter and model used to list attributes elsewhere.

To test:

- Navigate to the add attribute screen and verify everything still works

Update release notes:
